lib/libproc/proc_util.c

```diff
@@ -180,8 +181,15 @@ proc_getwstat(struct proc_handle *phdl)
 char *
 proc_signame(int sig, char *name, size_t namesz)
 {
+	char buf[SIG2STR_MAX + 3];
 
-	strlcpy(name, strsignal(sig), namesz);
+	/* sig2str() omits the leading "SIG" */
+	(void)strlcpy(buf, "SIG", sizeof(buf));
+
+	if (sig2str(sig, buf + 3) == 0)
+		(void)strlcpy(name, buf, namesz);
```

Comment:
Wouldn't it be simpler to just `snprintf(name, namesz, "SIG%s", buf)` here... no need for `+ 3` or `strlcpy()` above.

Reply:
> Wouldn't it be simpler to just `snprintf(name, namesz, "SIG%s", buf)` here... no need for `+ 3` or `strlcpy()` above.

Agree.  It makes no sense to follow their style here.  Fixed.